redist/modifier.py

```diff
@@ -479,7 +479,7 @@ def map(target_samples, kinematic_samples, target_bins, kinematic_bins):
     """
```

Comment:
@TimWue02 can you add the weights as argument here:
```
weights (array, optional): Weights for individual samples.
```
Everything else looks good!